test/e2e/kubelet_perf.go

```diff
@@ -95,15 +95,53 @@ func runResourceTrackingTest(framework *Framework, podsPerNode int, nodeNames se
 	By("Reporting overall resource usage")
 	logPodsOnNodes(framework.Client, nodeNames.List())
 	rm.LogLatest()
-
-	summary := rm.GetCPUSummary()
-	Logf("%s", rm.FormatCPUSummary(summary))
-	verifyCPULimits(expected, summary)
+	usageSummary, err := rm.GetLatest()
+	Expect(err).NotTo(HaveOccurred())
+	Logf("%s", rm.FormatResourceUsage(usageSummary))
+	// TODO(yujuhong): Set realistic values after gathering enough data.
+	verifyMemoryLimits(resourceUsagePerContainer{
+		"/kubelet":       &containerResourceUsage{MemoryRSSInBytes: 500 * 1024 * 1024},
```

Comment:
Just checking if you plan on resolving that TODO anytime soon ;)

On Fri, Feb 12, 2016 at 1:09 PM, Yu-Ju Hong <notifications@github.com>
wrote:

> In test/e2e/kubelet_perf.go
> <https://github.com/kubernetes/kubernetes/pull/21020#discussion_r52798566>
> :
>
> > @@ -95,15 +95,53 @@ func runResourceTrackingTest(framework *Framework, podsPerNode int, nodeNames se
> >  	By("Reporting overall resource usage")
> >  	logPodsOnNodes(framework.Client, nodeNames.List())
> >  	rm.LogLatest()
> > -
> > -	summary := rm.GetCPUSummary()
> > -	Logf("%s", rm.FormatCPUSummary(summary))
> > -	verifyCPULimits(expected, summary)
> > +	usageSummary, err := rm.GetLatest()
> > +	Expect(err).NotTo(HaveOccurred())
> > +	Logf("%s", rm.FormatResourceUsage(usageSummary))
> > +	// TODO(yujuhong): Set realistic values after gathering enough data.
> > +	verifyMemoryLimits(resourceUsagePerContainer{
> > +		"/kubelet":       &containerResourceUsage{MemoryRSSInBytes: 500 * 1024 * 1024},
>
> See TODO two lines above
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/21020/files#r52798566>.
>